### PR TITLE
doc(baseConfig): refreshTokenSubmitProp and refreshTokenProp

### PR DESCRIPTION
**Pull request by cmerot**
added doc for refreshTokenSubmitProp and updated section about refreshTokenProp to reflect the code

### CODE REVIEW

File: doc/baseConfig.md [hunk 1]
```diff
@@ -97,19 +100,11 @@ refreshTokenName = 'token';
...
 
-
-// Id Token Options
-// =====================
-
-// The property from which to get the id token after a successful login
+// The property name from which to get the user authentication token. Can also be dotted idTokenProp.idTokenName
 idTokenProp = 'id_token';
-
-// If the property defined by `idTokenProp` is an object:
```
Comment (RWOverdijk) [inline doc/baseConfig.md:107]: Why did you remove these segmenting docs?

**Review by RWOverdijk — CHANGES_REQUESTED** [review]

**Comment by doktordirk** [comment]
easier to resubmit
